grype/db/internal/gormadapter/open.go

```diff
@@ -13,6 +13,8 @@ var writerStatements = []string{
 	// on my box it reduces the time to write from 10 minutes to 10 seconds (with ~1GB memory utilization spikes)
 	`PRAGMA synchronous = OFF`,
 	`PRAGMA journal_mode = MEMORY`,
+	`PRAGMA cache_size = 100000`,
+	`PRAGMA mmap_size = 268435456`, // 256 MB
```

Comment:
notes from sync review with @wagoodman:

Cache size:
* These are not required, but improve performance
* The db write is much faster if more of the db is cached in memory
mmap_size:
* when reading the database, use more RAM to go faster

These numbers are both higher than the defaults.